### ISSUE TITLE
(0.6) Fails to compile on Windows..

### ISSUE THREAD

**Issue by simast** [balthild/native-dialog-rs#24]
```
error[E0599]: no variant or associated item named `Windows` found for enum `RawWindowHandle` in the current scope
  --> E:\Dev\.cargo\registry\src\github.com-1ecc6299db9ec823\native-dialog-0.6.0\src\dialog_impl\win\message.rs:53:31
   |
53 |         Some(RawWindowHandle::Windows(handle)) => handle.hwnd as HWND,
   |                               ^^^^^^^ variant or associated item not found in `RawWindowHandle`
```

**Comment by i509VCB** [comment]
If you are using the latest winit, note that raw-window-handle has been upgraded to 0.4, meaning this crate needs to upgrade raw-window-handle

**Comment by balthild** [comment]
Sorry... I've just upgraded raw-window-handle yesterday, and have forgotten to `cargo check` with windows target. I'll fix this soon.

**Comment by balthild** [comment]
This should be fixed in v0.6.1.
